lib/ledger_sync/ledgers/netsuite/client.rb

```diff
@@ -63,7 +63,7 @@ def get(**keywords)
         def ledger_resource_path(args = {})
           resource = args.fetch(:resource, nil)
 
-          ret = resource.class.resource_type.to_s # This can be turned into a case statement if we need to override
+          ret = resource.class.resource_type.to_s.camelcase # This can be turned into a case statement if we need to override
```

Comment:
@ryanwjackson I did this as the api endpoint is `/customerDeposit`, whereas before it was hitting `/customer_deposit`, is this ok or is there a better way of doing this?

Reply:
As long as this works for other NS routes.  We also have the `ledger_resource_type` overrides class methods on the Client.  It just isn't used in this function.  We could use it here (see the QBO Client for an example).